### ISSUE TITLE
Export animations don't work correctly

### ISSUE THREAD

**Issue by KiborgMaster** [KhronosGroup/glTF-Blender-IO#891]
VIDEO: https://www.youtube.com/watch?v=QTlSDDOnAUI

Good day, sorry for my not entirely correct English, but I think in Wi-Fi you will understand everything. The model export with animation does not work, after export the animation does not work correctly, the only export in which this one works is https://github.com/KhronosGroup/glTF-Blender-Exporter but it only works in 2.79 blender, 3D model in it and it was created therefore in the new version, the textures were not detected automatically. And the very first bug is very interesting, the exportet works only after hiding the appearance of the model, exporting the void, and only then exporting the model.

**Comment by julienduroure** [comment]
Hello,
Without the .blend file, we can't do anything. Please follow the template (that you deleted to write your message). Without these information, we will not be able to help you.
We, at a minimum, need the blend file, and the exact version of Blender you are using.

Thanks !

**Comment by KiborgMaster** [comment]
Sorry, I attached a blender file. The blender version with which the model 2.79 was created. And now I'm using version 2.81

[QHERO2.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/4110742/QHERO2.zip)

**Comment by KiborgMaster** [comment]
I made reconciliations with the problems of the site exporter https://sketchfab.com, they are used by your exporter and they have problems because of a bug in the exporter. For the test, I downloaded the source code of the model from FBX from them and, having made an expert at GLTF, I got exactly the same problem about which they wrote to their support center, but it turns out you need to talk about it, in general, when you solve the first problem and I will show you the second with a specific misfortune.

**Comment by julienduroure** [comment]
This is a duplicate of #756 
With sample animation, TRS animation directly on the armature are not taken into account.

Because your bones don't have any constraints, you can export with "always sample animation" disabled, and animations seems to be OK.

Because of duplicate, closing this ticket. #756 stays the main ticket for this problem.
